django_dramatiq/apps.py

```diff
@@ -38,6 +39,7 @@ def ready(self):
         middleware = [load_middleware(path) for path in broker_settings.get("MIDDLEWARE", [])]
         broker = broker_class(middleware=middleware, **broker_options)
         dramatiq.set_broker(broker)
+        dramatiq.set_encoder(type(self).select_encoder())
```

Comment:
The `type(self)` is unnecessary. Classmethods correctly pass the appropriate object to the first positional argument based on the referencing context (https://docs.python.org/3/howto/descriptor.html#static-methods-and-class-methods).

Reply:
Excuse me but I follow you from this line[0].
Do you want to fix this line too?

[0] https://github.com/Bogdanp/django_dramatiq/blob/master/django_dramatiq/apps.py#L34

Reply:
Hah! Yes, please do. :)